index.js

```diff
@@ -1,18 +1,29 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header");
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "blue";
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
-
-
+console.log('This is what the traveler object looks like: ', traveler)
+document.querySelector("div.traveler > div#profile > h2").textContent = traveler.name;
+document.querySelector("div.traveler > div#profile > em").textContent = traveler.nickname;
+document.querySelector("div.traveler > div#profile > img").src = traveler.photo;
+document.querySelector("div.traveler > div#profile > img").alt = "pic of raffy";
```

Comment:
This works but you could also save the div#profile to a variable and then run a querySelector on that div
profielDiv.querySelector('h2')